independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/api/QuarkusRestClientProperties.java

```diff
@@ -7,4 +7,9 @@
      */
     public static final String MAX_REDIRECTS = "io.quarkus.rest.client.max-redirects";
     public static final String READ_TIMEOUT = "io.quarkus.rest.client.read-timeout";
+
+    /**
+     * See {@link io.netty.handler.codec.http.multipart.HttpPostRequestEncoder.EncoderMode}, RFC1738 by default
+     */
+    public static final String MULTIPART_ENCODER_MODE = "io.quarkus.rest.client.multipart-post-encoder-mode";
```

Comment:
We will definitely need to document this

Reply:
done